bin/post_peak_calling_processing.r

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env Rscript
 
 # R scripts for processing MACS output files (.xls)
-# Version 1.0
+# Version 1.2
```

Comment:
I'm not sure that we need a separate version for this script..? It will always be bundled with the pipeline, which has its own version numbers. I think it may be less confusing if we just use those?

Reply:
Fixed.